src/PointwiseFunctions/ScalarTensor/SourceTags.hpp

```diff
@@ -0,0 +1,47 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Options/String.hpp"
+
+/// \cond
+namespace ScalarTensor::OptionTags {
+struct Group;
+}  // namespace ScalarTensor::OptionTags
+/// \endcond
+
+namespace ScalarTensor {
+namespace OptionTags {
+/*!
+ * \brief Start and duration time for ramp up function.
+ */
+struct RampUpParameters {
```

Comment:
I would make two different option tags for this. One `RampUpStart` and the other `RampUpDuration`. As it currently stands, if I see `RampUpParameters` in an input file, I'll have no clue what they mean (and I definitely won't 2 years from now)